docs/installation.rst

```diff
@@ -24,29 +24,15 @@ Verifying your Installation
 ---------------------------
 To check if the installation was successful, you can run::
 
-    python -c 'import cassandra; print cassandra.__version__'
+    python -c 'import scylla; print scylla.__version__'
```

Comment:
this should be change, the internal module name is still cassandra
i.e. people shouldn't need to change their code moving to the fork